.github/workflows/check-formatting.yml

```diff
@@ -0,0 +1,19 @@
+name: Check formatting
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
```suggestion
      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
```

Reply:
Oh, I hoped that the update bot would take care of that quickly after merge…

Are you able to suggest a tool that does that automatically, such that I do not have to look up the git rev ID manually?